app/backend/lib/sow_import/tab_2.ts

```diff
@@ -0,0 +1,78 @@
+import * as XLSX from 'xlsx';
+import { performQuery } from '../graphql';
+
+const createSomeMutation = `
+  mutation tab2Mutation($input: SowTab2Input!) {
+    createSowTab2(input: {sowTab2: $input}) {
+      clientMutationId
+    }
+  }
+`;
+
+const readData = async(wb, sheet_name) => {
+  const sheet = XLSX.utils.sheet_to_json(wb.Sheets[sheet_name], { header: "A" });
+  const result = [];
+  let tableDetected = false;
+  for (let row = 1; row < sheet.length; row ++) {
+    let validLine = false;
+      const suspect = sheet[row]['A'];
+      let value;
+      if (suspect === undefined) continue;
+      if (typeof(suspect) !== 'string') { 
+        value = suspect.toString();
+      }
+      else {
+        value = suspect;
+      }
+      // if (typeof(value) !== 'string') continue;
+      if (value.indexOf('Entry Number')>-1) {
+          tableDetected=true;
+          continue;
+      }  
+      if (tableDetected) { 
+          validLine = sheet[row]['O'] && typeof(sheet[row]['O']) === 'string'; 
+          value = sheet[row]['O'];
+          // now reading line
+          if (validLine && (value === 'Complete')) {
+            const lineData = 
+            {  
+              entryNumber: sheet[row]['A'], 
+              projectSiteName: sheet[row]['B'], 
```

Comment:
Something to check with @elliottSlee, part of the acceptance criteria is that any data that refers to the same json key in the application form data should be the same

Reply:
yes, #1533 mentions other fields, nothing about projectSiteName

Reply:
Sorry projectSiteName was just a random one I decided to use for commenting on this part here. I'm also not sure here either if any of these are duplicated in the application data though. Probably not but wouldn't be a bad idea checking in with Elliot